config/settings_schema.json

```diff
@@ -516,21 +480,17 @@
         "type": "range",
         "id": "inputs_shadow_blur_radius",
         "min": 0,
-        "max": 20,
-        "step": 2,
+        "max": 40,
+        "step": 5,
```

Comment:
I'm wondering if the steps here are too large for blur radius.  I feel like each `px` you add to a blur radius adds a lot.  Not a strongly held opinion, so I defer to @melissaperreault and @KaichenWang on the decision.